itext2kg/__init__.py

```
@@ -1,7 +1,9 @@
 from .documents_distiller import DocumentsDistiller
+from .documents_distiller import DocumentsDistiller as DocumentDistiller  # Add alias for backward compatibility
 from .graph_integration import Neo4jStorage as GraphIntegrator
+from .graph_integration import Neo4jStorage, FalkorDBStorage  # Add both integrations
 from .itext2kg import iText2KG
 from .itext2kg_star import iText2KG_Star
 from . import logging_config 
```

Ruff (0.12.2)

`.logging_config` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)